static/app/components/feedback/feedbackItem/feedbackItemLoader.tsx

```diff
@@ -3,12 +3,15 @@ import FeedbackErrorDetails from 'sentry/components/feedback/details/feedbackErr
 import FeedbackItem from 'sentry/components/feedback/feedbackItem/feedbackItem';
 import useCurrentFeedbackId from 'sentry/components/feedback/useCurrentFeedbackId';
 import useFetchFeedbackData from 'sentry/components/feedback/useFetchFeedbackData';
+import useFetchFeedbackInfiniteListData from 'sentry/components/feedback/useFetchFeedbackInfiniteListData';
 import Placeholder from 'sentry/components/placeholder';
 import {t} from 'sentry/locale';
 
 export default function FeedbackItemLoader() {
   const feedbackId = useCurrentFeedbackId();
   const {issueResult, issueData, tags, eventData} = useFetchFeedbackData({feedbackId});
+  const {issues} = useFetchFeedbackInfiniteListData();
+  const inSearch = issues.find(issue => issue.id === feedbackId);
```

Comment:
m: I would call this `inFeedbackList` instead of search to make it a bit more clear that it's not always due to search